bridge/setu/processor/staking.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	defaultDelayDuration time.Duration = 10 * time.Second
+	defaultDelayDuration time.Duration = 15 * time.Second
```

Comment:
Could we create an issue to move such hard coded values to a config and pass the config either as a parameter or as a value in context?

Reply:
Sure, Will create an issue for this.